package.json

```diff
@@ -8,6 +8,7 @@
     "async": "^3.2.4",
     "fast-deep-equal": "^3.1.3",
     "hat": "0.0.3",
+    "inherits": "^2.0.4",
```

Comment:
If we don't want to add an extra dependency, we could potentially add something like `inherits` itself does?

```js
try {
  var util = require('util');
  if (typeof util.inherits !== 'function') throw new Error();
  module.exports = util.inherits;
} catch (e) {
  module.exports = require('inherits');
}
```

Reply:
I actually think this is better. Going to change to this.